src/node_options.cc

```diff
@@ -619,6 +619,9 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
   AddOption("--test-timeout",
             "specify test runner timeout",
             &EnvironmentOptions::test_runner_timeout);
+  AddOption("--experimental-minimum-coverage",
+            "specify minimum code coverage threshold value",
+            &EnvironmentOptions::test_minimum_coverage);
```

Comment:
I think this should also be added in the following file as well:
https://github.com/nodejs/node/blob/01719333623aaa324d05011ffcba0dddc0ea3666/doc/node.1#L183-L184